src/main/java/gregicadditions/recipes/MachineCraftingRecipes.java

```diff
@@ -242,7 +139,7 @@ public static void init() {
 				ModHandler.addShapedRecipe("stainless_steel_drum", GATileEntities.STAINLESS_STEEL_DRUM.getStackForm(), " h ", "PRP", "PRP", 'P', "plateCurvedStainlessSteel", 'R', "stickLongStainlessSteel");
 				ModHandler.addShapedRecipe("titanium_drum", GATileEntities.TITANIUM_DRUM.getStackForm(), " h ", "PRP", "PRP", 'P', "plateCurvedTitanium", 'R', "stickLongTitanium");
 				ModHandler.addShapedRecipe("tungstensteel_drum", GATileEntities.TUNGSTENSTEEL_DRUM.getStackForm(), " h ", "PRP", "PRP", 'P', "plateCurvedTungstenSteel", 'R', "stickLongTungstenSteel");
-			} else if (!(GAConfig.GT6.BendingCurvedPlates || GAConfig.GT6.BendingCylinders)) {
```

Comment:
The logic here is redundant, as if the curved plates are not registered, then regular plates should just be used. In addition, it can be seen that before the final line was added on the initial if, checking if the curved plates were registered, that the conditionals on this `else if` were basically inverted from the conditionals on the if.